blocks/api/post.pegjs

```diff
@@ -91,11 +140,11 @@ Special_Chars
 Newline
   = [\r\n]
 
+Whitespace
+  = [\s\r\n]+
```

Comment:
actually no they weren't included. I'm not sure what `\s` actually gave us. I have updated with the proper definition of "whitespace" from JS though in 8fde722c. we may want to throttle back on this too since it includes things like Unicode whitespace characters as well